extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,89 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import StringBuilder from 'ad-engine/src/utils/string-builder';
+import config from './context';
+import Client from "ad-engine/src/utils/client";
+
+Context.extend(config);
+let supportedTemplates = [ BigFancyAdBelow ];
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+function init(slotRegistry, pageLevelTargeting, skin) {
+	overrideSlotService(slotRegistry);
+	updatePageLevelTargeting(pageLevelTargeting, skin);
+}
+
+function overrideSlotService(slotRegistry) {
+	SlotService.getBySlotName = (id) => {
+		if (id) {
+			let slot = slotRegistry.get(id);
+			return unifySlotInterface(slot);
+		}
+	};
+}
+
+function unifySlotInterface(slot) {
+	const slotContext = Context.get(`slots.${slot.name}`) || {targeting: {}};
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this ES6 variable declaration with var. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AConstLet)

[project-id]: wikia:app